Sources/BraveWallet/Crypto/Portfolio/AddCustomAssetView.swift

```diff
@@ -109,21 +109,20 @@ struct AddCustomAssetView: View {
         ) {
           TextField(Strings.Wallet.enterAddress, text: $addressInput)
             .onChange(of: addressInput) { newValue in
-              if !newValue.isEmpty, newValue.isETHAddress {
-                userAssetStore.tokenInfo(by: newValue) { token in
-                  if let token = token {
-                    if nameInput.isEmpty {
-                      nameInput = token.name
-                    }
-                    if symbolInput.isEmpty {
-                      symbolInput = token.symbol
-                    }
-                    if !token.isErc721 {
-                      if decimalsInput.isEmpty {
-                        decimalsInput = "\(token.decimals)"
-                      }
-                    }
-                  }
+              guard !newValue.isEmpty else { return }
+              userAssetStore.tokenInfo(address: newValue) { token in
+                guard let token else { return }
+                if nameInput.isEmpty {
+                  nameInput = token.name
+                }
+                if symbolInput.isEmpty {
+                  symbolInput = token.symbol
+                }
+                if !token.isErc721, decimalsInput.isEmpty {
```

Comment:
need to add `!token.isNft`. probably missed it originally

Reply:
Thanks, fixed in [e65de9a](https://github.com/brave/brave-ios/pull/6990/commits/e65de9a188b5ba8dd794d3b4476483ec0b644a01)